src/js/View.js

```diff
@@ -621,7 +621,7 @@ View = (function() {
                         }
                     }
                 }
-                if (!hasTouchEvents) {
+                if (e.type === "mousemove") {
```

Comment:
I'm not entirely sure what the reason behind this change was. Reading it now, it seems like the behaviour would be exactly the same with or without this change